cockatrice/src/chatview.cpp

```diff
@@ -168,8 +168,17 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
     // use different color for server messages 
     defaultFormat = QTextCharFormat();
     if (sender.isEmpty()) {
-        defaultFormat.setForeground(Qt::darkGreen);
-        defaultFormat.setFontWeight(QFont::Bold);
+        switch (messageType) {
+            case 1:
```

Comment:
what does 0, 1, 2 means? Please create an enum with these values so that we can use labels instead of magic numbers.